
＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,650                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15004]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,659                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15014]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,660                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15014]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,660                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15015]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,660                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15015]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,661                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15016]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,662                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15016]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,662                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15016]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,662                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15017]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,663                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15017]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,663                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15018]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,664                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15018]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,664                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15018]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,664                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15019]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,665                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15019]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,665                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15019]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,665                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15020]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,666                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15020]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,666                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15020]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,666                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15021]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,667                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15021]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,667                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15022]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,667                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15022]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,668                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15022]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,668                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15023]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,668                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15023]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,669                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15023]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,669                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15024]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,670                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15024]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,670                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15024]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,670                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15025]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,671                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15025]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,671                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15025]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,671                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15026]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,672                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15026]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,672                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15026]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,672                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15027]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,673                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15027]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,673                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15028]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,673                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15028]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,674                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15028]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,674                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15029]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,674                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15029]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,675                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15029]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,675                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15030]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,676                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15030]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,676                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15030]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,676                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15031]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,677                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15031]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,677                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15032]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,678                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15032]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,678                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15032]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,678                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15033]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,679                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15033]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,679                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15034]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,679                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15034]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,680                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15034]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,680                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15035]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,680                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15035]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,681                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15035]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,681                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15036]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,682                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15036]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,682                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15036]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,682                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15037]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,683                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15037]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,683                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15037]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,683                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15038]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,684                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15038]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,684                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15039]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,684                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15039]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,685                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15039]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,685                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15040]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,686                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15040]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,686                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15040]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,686                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15041]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,687                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15041]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,687                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15041]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,687                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15042]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,688                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15042]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,688                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15042]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,688                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15043]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,689                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15043]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,689                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15043]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,689                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15044]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,690                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15044]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,690                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15044]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,690                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15045]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,313                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14668]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,691                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15046]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,691                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15046]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,692                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15046]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,692                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15047]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,693                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15047]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,693                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15048]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,693                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15048]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,694                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15048]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,694                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15049]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,694                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15049]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,696                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15051]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,697                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15051]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,697                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15052]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,698                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15052]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,698                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15052]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,037                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[14392]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,699                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15054]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,700                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15054]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,701                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15055]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,701                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15056]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,702                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15057]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,703                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15057]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,703                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15058]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,704                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15059]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,705                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15059]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26
